sdb/__init__.py

```diff
@@ -177,3 +177,53 @@ def get_first_type(objs: Iterable[drgn.Object]
     if first_obj is None:
         return None, []
     return first_obj.type_, itertools.chain([first_obj], iterator)
+
+
+def type_canonicalize(t: drgn.Type) -> drgn.Type:
```

Comment:
I just realized this...now with the reorganization of files you probably want to add these functions in `sdb/target.py` and the in `sdb/__init__.py` apply the following change:
```
from sdb.target import ....., type_canonicalize_name, ...
```
Let me know if you have any questions on this